### PR TITLE
[FEATURE] unsafeOptimisticTransport

### PR DESCRIPTION
**Pull request by sentientwaffle**
cc: @emschwartz

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -90,12 +92,13 @@ function sendPayment (params) {
...
  *   If not provided, one will be generated.
  * @param {String} [params.caseId] = A notary case ID - if not provided, one will be generated
+ * @param {Boolean} [params.unsafeOptimisticTransport] - Set for optimistic mode
  * @param {String|Buffer} [params.ca] - Optional TLS CA if not using default CA (optional for https requests)
  */
 function executePayment (sourceTransfer, params) {
   return co(function * () {
     const isAtomic = !!params.notary
-    const isOptimistic = !params.receiptCondition && !isAtomic
+    const isOptimistic = !!params.unsafeOptimisticTransport
```
Comment (emschwartz) [inline src/index.js]: It should throw an error if you don't sent `unsafeOptimisticTransport` and don't provide a condition

Comment (sentientwaffle) [inline src/index.js]: I think it does: https://github.com/interledger/five-bells-sender/blob/9698876304d2455edd4fb417576a6a0f6044ad75/src/index.js#L142-L144

Comment (emschwartz) [inline src/index.js]: Sorry, missed that

**Comment by emschwartz** [comment]
LGTM

**Pull request merged**